apps/web/src/lang/fr.json

```
@@ -5003,6 +5027,10 @@
     "defaultMessage": "Expérience et compétences requises",
     "description": "Title for the experience and skills section of a pool advertisement"
   },
+  "VYzAZy": {
+    "defaultMessage": "Ce processus est à l'étape d’ébauche <heavyWarning> </heavyWarning> et n'a pas encore été annoncé.",
```

The original English string doesn't include the word stage (_étape_), but I am assuming it should also be included in the marked up copy.
```suggestion
    "defaultMessage": "Ce processus est à <heavyWarning>l'étape d’ébauche</heavyWarning> et n'a pas encore été annoncé.",
```